x-pack/platform/plugins/shared/fleet/server/services/epm/packages/install.ts

```diff
@@ -1290,126 +1283,6 @@ export async function ensurePackagesCompletedInstall(
   return installingPackages;
 }
 
-export async function installAssetsForInputPackagePolicy(opts: {
```

Comment:
I moved this function to a new file that groups the utilities needed for input-type packages